src/graph_notebook/notebooks/04-Machine-Learning/neptune_ml_utils.py

```diff
@@ -279,7 +289,7 @@ def __upload_to_s3(self, bucketname: str):
                     self.formatted_directory, file), bucket, f'{file_path}/{file}')
 
     def prepare_movielens_data(self, s3_bucket: str):
-        bucket_name = f'{s3_bucket}/neptune-formatted/movielens-100k'
```

Comment:
This data is not really ml specific other than that is what we used as a demo.  The format here is in CSV for Neptune not Neptune ML

Reply:
This should be good now, resolving but feel free to reopen if I am mistaken